bridges/centralized-ethereum/src/actors/eth_poller.rs

```diff
@@ -72,7 +72,12 @@ impl EthPoller {
                     contract::Options::default(),
                     None,
                 )
-                .await;
+                .await
+                .map_err(|err| {
+                    log::error!("Fail to read dr bytes from contract: {:?}", err.to_string());
```

Comment:
```suggestion
                    log::error!("Fail to read requestsCount from contract: {:?}", err.to_string());
```

Also, you don't need `"{:?}", err.to_string()`. You can use `"{}", err` to get the human readable error message, or `"{:?}", err` to get the debug representation.

Reply:
@mariocao I think you missed this suggestion